app/models/placement_group.rb

```diff
@@ -0,0 +1,113 @@
+class PlacementGroup < ApplicationRecord
+  acts_as_miq_taggable
+  include YAMLImportExportMixin
+  include SupportsFeatureMixin
+  include NewWithTypeStiMixin
+  def self.new(*args, &block)
+    if self == Authentication && (args.empty? || args.first.kind_of?(Hash))
+      args = [{}] if args.empty?
+      h = args.first
+
+      type = h[inheritance_column.to_sym] || h[inheritance_column.to_s]
+      h[inheritance_column.to_sym] = "AuthUseridPassword" if type.nil?
+    end
+
+    super
+  end
+
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::CloudManager"
+  has_many :vms, :dependent => :nullify
+
+  delegate :queue_name_for_ems_operations, :to => :ext_management_system, :allow_nil => true
+
+  supports_not :safe_delete
+
+  def supports_safe_delete
+    supports?(:safe_delete)
+  end
+
+
+
+  def self.class_by_ems(ext_management_system)
+    ext_management_system&.class_by_ems(:CloudVolume)
+  end
+
+  def self.my_zone(ems)
+    # TODO(pblaho): find unified way how to do that
+    ems ? ems.my_zone : MiqServer.my_zone
+  end
+
+  def my_zone
+    self.class.my_zone(ext_management_system)
+  end
+
+  def self.create_placement_group_queue(userid, ext_management_system, options = {})
+  end
+
+  def self.create_placement_group(ems_id, options = {})
+    raise ArgumentError, _("ems_id cannot be nil") if ems_id.nil?
+    ext_management_system = ExtManagementSystem.find(ems_id)
+    raise ArgumentError, _("ext_management_system cannot be found") if ext_management_system.nil?
+
+    klass = ext_management_system.class_by_ems(:Placement)
+    klass.raw_create_placement_group(ext_management_system, options)
+  end
+
+  def self.raw_create_placement_group(_ext_management_system, _options = {})
+    raise NotImplementedError, _("raw_create_placement_group must be implemented in a subclass")
+  end
+
+  def delete_placement_group(userid)
+    task_opts = {
+      :action => "deleting Placement Group for user #{userid}",
+      :userid => userid
+    }
+
+    queue_opts = {
+      :class_name  => self.class.name,
+      :method_name => 'delete_placement_group',
+      :instance_id => id,
+      :role        => 'ems_operations',
+      :queue_name  => ext_management_system.queue_name_for_ems_operations,
+      :zone        => ext_management_system.my_zone,
+      :args        => []
+    }
+
+    MiqTask.generic_action_with_callback(task_opts, queue_opts)
+  end
+
+  def delete_placement_group
+    raw_delete_placement_group
+  end
+
+  def raw_delete_placement_group
+    raise NotImplementedError, _("raw_delete_placement_group must be implemented in a subclass")
+  end
```

Comment:
If you have an Operations module, shouldn't these live there?  I actually don't have an issue with create/update/delete being in the class but weird to have an Operations module and not use it

Reply:
As advised I have removed the Operations module.